cent/types/override.py

```diff
@@ -0,0 +1,18 @@
+from typing import Optional
+
+from cent.types.base import CentObject
+
+
+class Override(CentObject):
+    """Override object."""
+
+    presence: Optional[bool] = None
```

Comment:
the definition is not just `bool` - but [BoolValue](https://github.com/centrifugal/centrifugo/blob/139e3085a95f7290be12a03e4310d07123bc5fb4/internal/apiproto/api.proto#L164) - same for other fields